torch_xla/csrc/passes/remove_unused_forward_outputs.cpp

```diff
@@ -1,4 +1,5 @@
 #include "remove_unused_forward_outputs.h"
```

Comment:
You can do it in a follow up, but I would like those passes to use TF_VLOG() to dump their activity, *and* the input/output graph, like we do in XLA.